src/test/java/com/solvd/carina/orangehrm/web/AbstractWebTest.java

```diff
@@ -0,0 +1,18 @@
+package com.solvd.carina.orangehrm.web;
+
+import com.solvd.carina.orangehrm.pages.DashboardPage;
+import com.solvd.carina.orangehrm.pages.EmployeeTimeSheetsPage;
+import com.solvd.carina.utils.LoginUtils;
+import com.solvd.carina.utils.NavigationUtils;
+import com.zebrunner.carina.core.IAbstractTest;
+
+public class AbstractWebTest implements IAbstractTest {
+
+    public DashboardPage login() {
+        return new LoginUtils().loginStandardUser();
+    }
+
+    public EmployeeTimeSheetsPage navigateToTimeSheets() {
+        return new NavigationUtils().navigateToEmployeeTimeSheetsPage();
+    }
```

Comment:
I think you can use utils directly in the test. Here in AbstractWebTest you can create instances of your utils and use it in your tests
```suggestion
   LoginUtils loginUtils = new LoginUtils();
   
   NavigationUtils navigationUtils = new NavigationUtils();
```